lua/ulx/modules/sh/pcrestrict.lua

```diff
@@ -0,0 +1,45 @@
+CATEGORY_NAME = "CFC"
```

Comment:
with the cfc ulx repository we started putting ulx modules in categories representing what they do instead of just "CFC" e.g. `Cleanup`
maybe we should do that here